src/config/dynamic-theme-fixes.config

```diff
@@ -10567,16 +10582,6 @@ img[src*="bookmark.png"]
 
 ================================
 
-goproblems.com
```

Comment:
Removing other fixes won't be accepted to merge for sure!

Reply:
That was unintended for sure!